### ISSUE TITLE
our story - external links open in same tab

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#107]
Clicking on the below external links like "Our Founder" opens the web page in the same tab; however the icon next to it indicates it should be opening in a new one.

![Screen Shot 2020-10-13 at 10 31 29 AM](https://user-images.githubusercontent.com/618307/95874956-79a6b000-0d3f-11eb-9ef0-835f9ca83695.png)

**Comment by kimisgold** [comment]
This should be fixed and ready for testing.
